README.md

```diff
@@ -1,5 +1,14 @@
 # near-api
-The near-api is a simple Rust library that helps developers interact easily with the NEAR blockchain. The library was highly inspired by the API of the [near-cli-rs](https://github.com/near/near-cli-rs) library. The library extensively utilizes builder patterns, this way we guide the users through the user flow, preventing most of the errors and focusing on each step.
+<p>
+    <a href="https://docs.rs/near-api"><img src="https://docs.rs/near-api/badge.svg?style=flat-square" alt="Reference Documentation" /></a>
+    <a href="https://blog.rust-lang.org/2024/09/05/Rust-1.81.0.html"><img src="https://img.shields.io/badge/rustc-1.81+-lightgray.svg?style=flat-square" alt="MSRV" /></a>
```

Comment:
the badge thing could be replaced with a reference to `rust-version = ...` line in manifest, and thus be always +/- autoupdated . 
the badge thingy is regularly forgotten to be updated, so there's little point in it, if it's good and readable, but not true